cables.py

```diff
@@ -439,12 +435,9 @@ def find_style_tag_value(text,tag):
     """
     Search tags from in a string, return variables behind '='.
     """
-    if not none_or_empty(text):
-        array =text.split(';')
-        for x in array:
-            if x.find(tag) != -1:
-                parts = x.partition('=')
-                return parts[2]
+    match = re.search(f"{tag}(.+?);",text)
+    match = match.groups(1) if match else -1
+    return match
```

Comment:
Unit tests would help me remember all cases... I should add unit testing.

Reply:
this change improves performance with an order of magnitude! the rest is not so important